paramak/parametric_components/tokamak_plasma_from_points.py

```diff
@@ -0,0 +1,73 @@
+import math
+from pathlib import Path
+
+import numpy as np
+import scipy
+
+from paramak import Plasma
+
+
+class Plasma(Plasma):
```

Comment:
Can a user choose between this Plasma (from points) and the inherited class Plasma (from parameters) ?

Reply:
I need to finish, this PR but I like your plan of having one Plasma then two others plasma_form_points and plasma_from_parameters. I should rename this one